ProjectSetup/Contracts/V1/IUserRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System.Threading.Tasks;
+using ProjectSetup.Contracts.V1.Requests;
+
+namespace ProjectSetup.Contracts.V1
```

Comment:
Use contracts only for contact with the external world. You can place this into Repositories/Interfaces folder. Use the same logic for other Interfaces files in this folder.